examples/seismic/tti/tti_example2D.py

```diff
@@ -44,17 +41,16 @@ def source(t, f0):
 location = np.zeros((1, 2))
 location[0, 0] = origin[0] + dimensions[0] * spacing[0] * 0.5
 location[0, 1] = origin[1] + 2 * spacing[1]
-src.set_receiver_pos(location)
-src.set_shape(nt, 1)
-src.set_traces(time_series)
+src = PointSource(name='src', data=time_series, coordinates=location)
 
 # Receiver geometry
 receiver_coords = np.zeros((101, 2))
 receiver_coords[:, 0] = np.linspace(0, origin[0] +
                                     dimensions[0] * spacing[0], num=101)
 receiver_coords[:, 1] = location[0, 1]
-data.set_receiver_pos(receiver_coords)
-data.set_shape(nt, 101)
+rec = Receiver(name='rec', ntime=nt, coordinates=receiver_coords)
 
-TTI = TTI_cg(model, data, src, t_order=2, s_order=spc_order)
-rec, u, v, gflopss, oi, timings = TTI.Forward()
+solver = AnisotropicWaveSolver(model, source=src, receiver=rec,
+                               time_order=time_order,
+                               space_order=space_order)
+rec, u, v, summary = solver.forward()
```

Comment:
This whole ``test_ttiexample2D.py`` should have a structure similar to ``test_ttiexample3D.py``, where you have ``def source`` ``def setup`` ``def run``